provider/pkg/schema/gen.go

```diff
@@ -983,6 +994,11 @@ func (ctx *cfSchemaContext) updateDesc(refName, propName string, spec *jsschema.
 				}
 			}
 		}
+	} else if _, ok := forceDocumentationAugmentation[GetFullyQualifiedPropertyName(refName, propName)]; ok {
+		if desc, found := ctx.docs.GetPropertyDesc(refName, propName); found {
```

Comment:
Else: warn or report if augmentation was configured but not found?

Reply:
Yeah that's a good idea! I'm adding it to the report